src/index.js

```diff
@@ -161,10 +161,13 @@ class Client {
     }
 
     const message = await this.send(request).first()
-    const response = Response.decode(message)
+    if ((message || '') === '') {
```

Comment:
Why not validating if we got a message back?

```suggestion
    if (!message) {
```